pkgs/by-name/go/gopher64/package.nix

```diff
@@ -0,0 +1,74 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  makeWrapper,
+
+  bzip2,
+  sdl3,
+  zstd,
+
+  libxkbcommon,
+  vulkan-loader,
+  wayland,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gopher64";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "gopher64";
+    repo = "gopher64";
+    tag = "v${version}";
+    hash = "sha256-mDjqVQD4Vms1fSCGgCAWKL5LKRY36tH7Vq4NSnIVDCs=";
+    fetchSubmodules = true;
+  };
+
+  cargoPatches = [
+    # upstream rebuilds SDL3 from source
+    # this patch makes it use the SDL3 library provided by nixpkgs
+    ./use-sdl3-via-pkg-config.patch
+  ];
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-PSXzeyUkCnczEjKA98xpWSXuLp+MULyCi85CZYuitqg=";
+
+  env.ZSTD_SYS_USE_PKG_CONFIG = true;
+
+  nativeBuildInputs =
+    [
+      pkg-config
+      rustPlatform.bindgenHook
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      makeWrapper
```

Comment:
Where is that used?

Reply:
True, I have replaced that with patchelf. Will remove.